ci/github/unit-testing/webserver.bash

```diff
@@ -9,10 +9,28 @@ install() {
     pip list -v
 }
 
-test() {
+test_isolated() {
     pytest --cov=simcore_service_webserver --durations=10 --cov-append \
           --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
-          -v -m "not travis" services/web/server/tests/unit
+          -v -m "not travis" services/web/server/tests/unit/isolated
+}
+
+test_with_db_slow() {
+    pytest --cov=simcore_service_webserver --durations=10 --cov-append \
+          --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
+          -v -m "not travis" services/web/server/tests/unit/with_dbs/slow
+}
+
+test_with_db_medium() {
+    pytest --cov=simcore_service_webserver --durations=10 --cov-append \
+          --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
+          -v -m "not travis" services/web/server/tests/unit/with_dbs/medium
+}
+
+test_with_db_fast() {
+    pytest --cov=simcore_service_webserver --durations=10 --cov-append \
+          --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
+          -v -m "not travis" services/web/server/tests/unit/with_dbs/fast
```

Comment:
Could you please

Reply:
Could you please explain more what is the meaning of the following:
- isolated
- db slow
- db medium
- db fast

It would also be great to see a compressed version in the CI.